src/main/java/org/folio/circulation/resources/DeclareLostResource.java

```diff
@@ -70,10 +72,14 @@ private void declareLost(RoutingContext routingContext) {
   private CompletableFuture<Result<Loan>> declareItemLost(DeclareItemLostRequest request,
     Clients clients, WebContext context) {
 
-    final LoanRepository loanRepository = new LoanRepository(clients);
-    final ItemRepository itemRepository = new ItemRepository(clients, true, true, true);
-    final StoreLoanAndItem storeLoanAndItem = new StoreLoanAndItem(loanRepository, itemRepository);
-    final LostItemFeeChargingService lostItemFeeService = new LostItemFeeChargingService(clients);
+    final var itemRepository = new ItemRepository(clients);
+    final UserRepository userRepository = new UserRepository(clients);
+    final var loanRepository = new LoanRepository(clients, itemRepository,
+      new UserRepository(clients));
```

Comment:
Is there a reason this new statement isn't placed with the other one, farther down?

Reply:
I think I've resolved this duplication of instances